packages/zstandard/test_zstandard.py

```diff
@@ -0,0 +1,129 @@
+import os
+
+# Ensure C extension is used, and then CFFI as fallback, error if neither
+os.environ["PYTHON_ZSTANDARD_IMPORT_POLICY"] = "default"
```

Comment:
This looks wrong to me b/c you want the environment variable set inside Pyodide. You'll want:
```py
import pytest


@pytest.fixture
def env(selenium):
   selenium.run_python(
      """
      import os
      os.environ["PYTHON_ZSTANDARD_IMPORT_POLICY"] = "default"
      """
   )
   yield
```
and then pass env as a fixture to all the tests.